Reemit.Decompiler.Clr/Metadata/MethodImplCodeTypeAttributes.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Reemit.Decompiler.Clr.Metadata;
+
+/// <summary>
+/// Based on 23.1.11 Flags for methods [MethodImplAttributes]
+/// 
+/// Note:
+/// Not decorated with FlagsAttribute as these are not
+/// flags in the .NET sense.
+/// </summary>
+public enum MethodImplCodeTypeAttributes : ushort
+{
+    IL = 0x0000,
+    Native = 0x0001,
+    OPTIL = 0x0002,
```

Comment:
Apparently, the alternative spelling for this member is `OptIL`. This will probably better match C# naming conventions.
<img width="671" alt="image" src="https://github.com/v0idzz/Reemit/assets/12448522/489a98e5-cc2c-4513-bfb1-a1c56ef9f906">

Reply:
Good catch, this one didn't sit well with me. Fixed.